Count_Inversion/Count_Inversion.c

```diff
@@ -0,0 +1,75 @@
+// Count inversion using merge sort
+
+#include <stdio.h>
+
+//merge two sorted arrays such that resultant array is also sorted
+int merge(int array[], int aux[], int low, int mid, int high)
+{
+    int x = low, y = mid, z = low, count = 0;
+
+    // checking till left and right half of merge sort
+    while((x <= mid-1) && (y <= high)) 
+    {
+        if(array[x] <= array[y]) 
+        {
+            aux[z++] = array[x++];
+        }
+        else
+        {
+            aux[z++] = array[y++];
+            count += mid-x;
+        }
+    }
+
+    // Copy remaining elements
+    while(x <= mid-1)
+    {
+        aux[z++] = array[x++];
+    }
+    while(y <= high) 
```

Comment:
Add a line above this.
It would be great to have space after `while`, `for`, `if` etc keywords.